eng/pipelines/aggregate-reports.yml

```diff
@@ -48,16 +48,28 @@ stages:
             goals: 'clean install'
 
         - task: Maven@3
-          displayName: 'Generate Reports'
+          displayName: 'Generate Aggregate JavaDocs'
           inputs:
             mavenPomFile: pom.xml
-            options: '$(DefaultOptions) -Dinclude-non-shipping-modules -Dgpg.skip -Paggregatereports'
+            options: '$(DefaultOptions) -T 2C -Dinclude-non-shipping-modules -DskipTests -Dgpg.skip -Dcheckstyle.skip=true -Dspotbugs.skip=true -Djacoco.skip=true -Drevapi.skip=true -Pjavadoc'
             mavenOptions: '$(MemoryOptions) $(LoggingOptions)'
             javaHomeOption: 'JDKVersion'
             jdkVersionOption: $(JavaBuildVersion)
             jdkArchitectureOption: 'x64'
             publishJUnitResults: false
-            goals: 'verify site:site site:stage'
+            goals: 'javadoc:aggregate'
+
+        - task: Maven@3
+          displayName: 'Generate Aggregate JaCoCo coverage report'
+          inputs:
+            mavenPomFile: pom.xml
+            options: '$(DefaultOptions) -T 2C -Dinclude-non-shipping-modules -Dgpg.skip -Drevapi.skip -Pjacoco -Pcoverage'
+            mavenOptions: '$(MemoryOptions) $(LoggingOptions)'
+            javaHomeOption: 'JDKVersion'
+            jdkVersionOption: $(JavaBuildVersion)
+            jdkArchitectureOption: 'x64'
+            publishJUnitResults: false
+            goals: 'verify jacoco:report-aggregate'
```

Comment:
If we're looking for optimal performance here, such as if we want to run this pipeline in CI, we'll want to change the goals here. Right now, running this will re-run the compile and test compile phases, also it may no be skipping some linting phases such as Checkstyle, RevApi, or Spotbugs.